### PR TITLE
feat: Remove spatial feature flag

### PR DESCRIPTION
**Pull request by tihuan**
⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️
DO NOT MERGE THIS PR UNTIL WE GET THE GREEN LIGHT 🚦
⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️

### CODE REVIEW

File: client/src/components/menubar/index.tsx [hunk 1]
```diff
@@ -278,25 +277,25 @@ const MenuBar = ({
                 data-testid="drawer"
               />
             </ButtonGroup>
-            {isDownload && (
```
Comment (tihuan) [inline client/src/components/menubar/index.tsx:281]: no more conditional rendering on the download button

File: client/src/components/menubar/style.ts [hunk 2]
```diff
@@ -5,14 +5,8 @@ import { FEATURES } from "../../util/featureFlags/features";
...
 const isTest = getFeatureFlag(FEATURES.TEST);
-const isDownload = getFeatureFlag(FEATURES.DOWNLOAD);
 
-let FIRST_VERTICAL_THRESHOLD_WIDTH_PX = 655;
-if (isTest && isDownload) {
-  FIRST_VERTICAL_THRESHOLD_WIDTH_PX = 705;
-} else if (isTest || isDownload) {
-  FIRST_VERTICAL_THRESHOLD_WIDTH_PX = 685;
-}
+const FIRST_VERTICAL_THRESHOLD_WIDTH_PX = isTest ? 705 : 685;
```
Comment (tihuan) [inline client/src/components/menubar/style.ts:9]: @rainandbare PTAL if my math is correct here lol I'm thinking that now we have removed `isDownload` feature flag,  it should mean that the default width has increased from 655 to 685. And then if `isTest` is `true`, we bump it up to 705?

Thank you!

Comment (rainandbare) [reply client/src/components/menubar/style.ts:9]: If there is only ever one potential variable button then we can do: 

```suggestion
const FIRST_VERTICAL_THRESHOLD_WIDTH_PX = isTest ? 685 : 655;
```

**Review by rainandbare — APPROVED** [review]
Looks good to me! Left a suggestion with the appropriate break points.

File: client/src/components/menubar/style.ts (hunk 2 above)
Comment (tihuan) [reply client/src/components/menubar/style.ts:9]: Chatting with Suzette about this in Slack at the moment! Will share the result here later 👍

Comment (tihuan) [reply client/src/components/menubar/style.ts:9]: Confirmed that the current implementation is correct! Thanks again for your help, Suzette 🎉 !

**Pull request merged**